src/System.Data.SqlClient/tests/ManualTests/ADO/ParametersTest/ParametersTest.cs

```diff
@@ -99,6 +99,8 @@ public static void CodeCoverageSqlClient()
         [Fact]
         public static void Test_WithEnumValue_ShouldInferToUnderlyingType()
         {
+            if (!DataTestUtility.AreConnStringsValid()) return;
```

Comment:
Would show up as passed.  May be worth logging something first, but since we basically ignore passing tests, a few extra is not a big deal.